src/SerEvaluator.cs

```diff
@@ -492,7 +476,7 @@ private void WaitForDataLoad(ActiveTask task, UserParameter parameter, Cancellat
             {
                 while (true)
                 {
-                    Thread.Sleep(500);
+                    Thread.Sleep(1000);
```

Comment:
why you have to wait 1 sec.

Reply:
@konne I need this time for the separate case to load a script in one app.
Here I have to wait until Qlik has completed the Dataload.
Then build the report.
For the standard case, is this code line not significant.

Reply:
@mossbeard thx. please add a comment to the code for the future